code/game/gamemodes/changeling/traitor_chan.dm

```diff
@@ -59,7 +59,8 @@
 
 /datum/game_mode/traitor/changeling/post_setup()
 	for(var/datum/mind/changeling in changelings)
-		changeling.add_antag_datum(/datum/antagonist/changeling)
+		var/datum/antagonist/changeling/new_antag = new(null, changelings) //pass the number of changelings we're generating so they can have competivite objectives.

```

Comment:
I try to keep the antag New signature clean, so admin tools don't have to worry about additional stuff